DashWallet/Sources/Models/Explore Dash/Services/DashSpend/CTX/CTXSpendEndpoint.swift

```diff
@@ -56,7 +56,7 @@ public enum CTXSpendEndpoint {
 extension CTXSpendEndpoint: TargetType, AccessTokenAuthorizable {
     public var authorizationType: Moya.AuthorizationType? {
         switch self {
-        case .login, .verifyEmail, .refreshToken:
+        case .login, .verifyEmail, .refreshToken, .getMerchant:
```

Comment:
This looks strange.  I would think that .getMerchant should be handled under `default:` this seems like it would break the functions related to getting merchant data (up to date discounts and enabled/disabled)

On Android, we have to login before being able to determine if a merchant is disabled.  What does .getMerchant return without authorization?